docs/en/sql-reference/functions/other-functions.md

```diff
@@ -2984,6 +2984,57 @@ Result:
 └─────────┘
 ```
 
+## partitionId
+
+Returns computed [partition](../../engines/table-engines/mergetree-family/custom-partitioning-key.md) IDs of its arguments.
+
+:::note
+This function is slow and should not be called for large amount of rows.
+:::
+
+**Syntax**
+
+```sql
+partitionId(x[, y, ...]);
+```
+
+**Arguments**
+
+- `x` — Column for which to return the partition ID.
+- `y, ...` — Remaining N columns for which to return the partition ID (optional).
+
+**Return Type**
+
+- Partition ID that the row belongs to. [String](../data-types/string.md).
+
+**Example**
+
+Query:
+
+```sql
+DROP TABLE IF EXISTS mt;
+CREATE TABLE mt (n UInt64) ENGINE=MergeTree ORDER BY n PARTITION BY n % 2;
+INSERT INTO mt SELECT * FROM numbers(10);
+SELECT partitionId(*) FROM mt;
+```
+Result:
```

Comment:
The example should be more specific to demonstrate the behavior of this function.

Reply:
@alexey-milovidov i've replaced it with a less abstract example. How does it look now?